src/homepageExperience/components/steps/cli/InitializeTokenClient.tsx

```diff
@@ -0,0 +1,137 @@
+// Libraries
+import React, {FC, useEffect, useMemo} from 'react'
+import {useDispatch, useSelector} from 'react-redux'
+
+// Actions
+import {
+  createAuthorization,
+  getAllResources,
+} from 'src/authorizations/actions/thunks'
+
+// Selectors
+import {getOrg} from 'src/organizations/selectors'
+import {getMe} from 'src/me/selectors'
+import {getAllTokensResources} from 'src/resources/selectors'
+
+// Helper Components
+import CodeSnippet from 'src/shared/components/CodeSnippet'
+
+// Utils
+import {allAccessPermissions} from 'src/authorizations/utils/permissions'
+import {event} from 'src/cloud/utils/reporting'
+
+// Types
+import {AppState, Authorization} from 'src/types'
+
+// Tokens stuff
+
+type OwnProps = {
+  wizardEventName: string
+  setTokenValue: (tokenValue: string) => void
+  tokenValue: string
+}
+
+const collator = new Intl.Collator(navigator.language || 'en-US')
+
+// Client stuff
+
+export const InitializeTokenClient: FC<OwnProps> = ({
```

Comment:
Is there a reason behind naming this component `InitializeTokenClient` and not `InitializeClient`?